automobile/simple-vehicles.md

```diff
@@ -0,0 +1,73 @@
+## Simple Vehicles
+
+### CarSimple
+
+For each model of [Car](#car), a 'simple' PROTO is present too. These simplified
+kinematic PROTO models are not based on a `Robot` node but on a `Solid` node, it is
+therefore not possible to add sensors or control them. They are made to represent
+non-moving parked vehicles or to be moved using a Supervisor because they are much
+faster to simulate than the normal PROTO models.
+
+```
+ PROTO CarSimple {
+   field       SFVec3f    translation             0 0.4 0
+   field       SFRotation rotation                0 1 0 0
+   field       SFColor    color                   0.0 0.25 0.65
+   field       MFColor    recognitionColors       [ 0.0 0.25 0.65, 0.1 0.1 0.1 ]
+   field       MFString   plate                   "textures/plate.jpg"
+   field       SFString   name                    "vehicle"
+   field       SFBool     wheelBoundingObject     FALSE
+ }
```

Comment:
I would emphasize the fact that `CarSimple` is not an actual PROTO but defines the common structure of all the 'simple' version of the car.